docker-compose.yml

```diff
@@ -26,13 +26,13 @@ services:
       - ${LOCAL_HOME}/sipi/server:/sipi/server:delegated
     networks:
       - knora-net
-    restart: unless-stopped
     environment:
       - SIPI_EXTERNAL_PROTOCOL=http
       - SIPI_EXTERNAL_HOSTNAME=localhost
       - SIPI_EXTERNAL_PORT=1024
       - SIPI_WEBAPI_HOSTNAME=api
       - SIPI_WEBAPI_PORT=3333
+    # entrypoint: [ "valgrind", "--leak-check=yes", "/sipi/sipi" ] ## uncomment to run SIPI under valgrind
```

Comment:
What is valgrind? In what cases do we use it?

Reply:
valgrind is for finding memory leaks in C++. This is for debugging Sipi while running inside our stack.